lib/providers/map_data.dart

```diff
@@ -12,6 +13,8 @@ class MapData extends ChangeNotifier {
   PanelController panelController = new PanelController();
   String controllerStarting;
   String controllerDestination;
+  double swapButtonTop = SizeConfig.safeBlockVertical * 66;
+  double locationButtonTop = SizeConfig.safeBlockVertical * 75;
```

Comment:
This is causing the failure of unit tests. I don't think SizeConfig should be referenced here. Instead make SizeConfig as a separate Provider. Having it in MapData creates coupling.

Reply:
Also note SizeConfig is using MediaQuery and so when testing MapData it requires an initialized SizeConfig and therefore a MediaQuery.

Reply:
Fixed